tests/common/runner.rs

```diff
@@ -45,35 +45,35 @@ pub fn test_execute_revm<S: Storage + Clone + Send + Sync>(storage: S, txs: Vec<
     );
 }
 
+fn encode_receipt_2718(tx_type: u8, tx_result: &PevmTxExecutionResult) -> Bytes {
```

Comment:
This can be done later when we add the `Chain` parameter, but this would be better put in `primitives` than here :thinking:?